hooks/lib/provisioning_wizard.rb

```diff
@@ -253,6 +255,32 @@ def timezone
     @timezone ||= current_system_timezone
   end
 
+  def register_host=(answer)
+    @register_host = answer
+    if ['true', 'True', 'TRUE', true].include?(@register_host)
+      say "<%= color('You may Configure your host for Updates via the RHSM', :good) %>"
+      say "<%= color('Please visit the following for more information', :good) %>"
+      say "<%= color('https://access.redhat.com/labs/registrationassistant/', :good) %>"
+      say "<%= color('You may also choose to so now with your credentials.. continue?', :good) %>"
```

Comment:
Assume missing a word, 'do':
"You may also choose to do so now with your credentials...continue?"